bigquery/google/cloud/bigquery/client.py

```diff
@@ -656,13 +654,6 @@ def query_rows(self, query, job_config=None, job_id=None, timeout=None):
             given timeout.
         """
         job_id = _make_job_id(job_id)
```

Comment:
It would be even better if query_rows were
```
job = self.query(...)
return job.result(...)
```

Reply:
Done.